docs/mods.md

```diff
@@ -344,26 +346,26 @@ timermult/modtimermult/globalmodtimermult
 
 clearall
 
-wireframe
-wireframegirth
-wireframewidth
+*wireframe
+*wireframegirth
+*wireframewidth
 
-tiny
-tinyx
-tinyy
-tinyz
+*tiny
+*tinyx
+*tinyy
+*tinyz
 
-confusionoffset
-confusionxoffset
-confusionyoffset
-confusionzoffset
+*confusionoffset
+*confusionxoffset
+*confusionyoffset
+*confusionzoffset
 
 confusion
 confusionx
 confusiony
 confusionz
 
-dark
+*dark
 
 stealth
```

Comment:
How sure are you about these?
I know stealth is column specific but unmarked.

Since I've found xmod and cmod and stealth incorrectly labelled, I'm going to have to go in the game and check each mod to check your work before I can allow these to be merged. In the mirin template docs, I would rather have incomplete information than incorrect information, so this set of changes may hold me back from merging until I have a while to go over these.

Reply:
using the notitg devlogs as my source
https://craftedcart.gitlab.io/notitg_docs/versions.html#id5
look for "dark" since it is literally the only mention of it on the entire page lol
> + Column specific dark (hide targets)

Reply:
![image](https://user-images.githubusercontent.com/6820998/235312877-b04f266e-faa8-4179-bec3-c70803c5feac.png)
I guess you missed stealth from the changelog then? I still want to go over all the changes manually to make sure there's not any more mistakes

Reply:
I looked for "column specific" in the find menu so I missed that

Reply:
I fixed it